portal/templates/redesign/teach_new/onboarding_school.html

```diff
@@ -0,0 +1,150 @@
+{% extends 'redesign/teach_new/base_registering.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{ block.super }}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block pageID %}id='homeWrapper'{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<script type="text/javascript">
+    function openForm(evt, formName) {
+        // Declare all variables
+        var i, tabcontent, tablinks;
+
+        // Get all elements with class="tabcontent" and hide them
+        tabcontent = document.getElementsByClassName("tabcontent");
+        for (i = 0; i < tabcontent.length; i++) {
+            tabcontent[i].style.display = "none";
+        }
+
+        // Get all elements with class="tablinks" and remove the class "active"
+        tablinks = document.getElementsByClassName("tablinks");
+        for (i = 0; i < tablinks.length; i++) {
+            tablinks[i].className = tablinks[i].className.replace(" active", "");
+        }
+
+        // Show the current tab, and add an "active" class to the link that opened the tab
+        document.getElementById(formName).style.display = "block";
```

Comment:
I would opt for a .show/.hide class, rather than display block/none.  Sometimes you want your thing to be showing, but display: block causes CSS pain.  Using show/hide leaves you free to decide what "show" means on a case-by-case basis without rewriting your JS each time.